app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <string name="app_name">Oppia</string>
+  <string name="nav_header_title">Sean</string>
+  <string name="nav_header_subtitle">14 Stories Completed | 3 Topics In Progress</string>
+  <string name="nav_header_desc">Navigation header</string>
+  <string name="menu_home">Home</string>
+  <string name="menu_help">Help</string>
+  <string name="admin_settings">Admin Settings</string>
+  <!-- TODO: Remove or change this placeholder text -->
+  <string name="drawer_open">Open</string>
+  <string name="drawer_close">Close</string>
+  <!-- TODO: Remove or change this placeholder text -->
```

Comment:
TODO's should have an issue or a developer name associated with them. So file an issue and then link that issue here.

Reply:
Removed